### PR TITLE
feat: Spell 23-04-2026 (DEV-1593)

### PR DESCRIPTION
**Pull request by supercontracts**
# 2026-04-23 Spark Spell

## Forum Post
https://forum.skyeco.com/t/april-23-2026-proposed-changes-to-spark-for-upcoming-spell/27831

## Dependencies on Sky Core spell
None

## Notes for Reviewers
None

## Conditions for Spell Deployment
- [ ] All spell-specific tests are passing.
- [ ] All E2E tests are passing.
- [ ] All newly deployed addresses have been reviewed and added to the spark-address-registry in a PR that has been merged.
- [ ] All newly deployed addresses have been added to the Immunefi bug bounty program.
- [ ] There is complete test coverage of all changes in the spell, including any relevant edge cases.
- [ ] At least three approvals have been given from reviewers.
- [ ] CODE_RABBIT.md comment has been posted, AI review performed.
- [ ] Octane review performed.

## Conditions for Spell Handoff to Sky
- [ ] All spells payloads have been deployed and verified on Etherscan.
- [ ] All spell-specific tests are passing (including spell bytecode verification).
- [ ] All E2E tests are passing.
- [ ] Spell caster is passing in CI.
- [ ] At least three approvals have been given from reviewers on deployed spell.
- [ ] Comment from at least three reviewers confirming address of mainnet spell payload.

## Conditions for Spell Merge
- [ ] Spell handoff performed to Sky team on Discord.
- [ ] Spell handoff performed to Sky team on Signal.
- [ ] Spell address confirmed to be correct in sky-ecosystem/spells-mainnet repo open PR through comments by at least three reviewers.
- [ ] Sky deployed spell confirmed through a comment in this PR to contain correct Spark payload address by at least three reviewers.

## Summary by CodeRabbit

* **New Features**
  * Added governance proposals (April 9 and April 23, 2026) with rate limit adjustments, deposit cap increases for Spark Savings, and interest rate model updates.
  * Configured rETH deprecation and treasury distributions via governance.

* **Documentation**
  * Added reserve configuration change documentation.

* **Tests**
  * Comprehensive test coverage for new governance proposals and configuration changes.

### CODE REVIEW

File: src/proposals/20260423/Spell_20260423.t.sol [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    function test_ETHEREUM_sparkLend_deprecateRETH() external onChain(ChainIdUtils.Ethereum()) {
+        SparkLendContext memory ctx = _getSparkLendContext();
+
+        ReserveConfig[] memory allConfigsBefore = _createConfigurationSnapshot('', ctx.pool);
+
+        ReserveConfig memory rethConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'rETH');
+
+        assertEq(rethConfigBefore.isFrozen, false);
+
+        _executeAllPayloadsAndBridges();
```
[the remark this answers is not in the document]
Comment (supercontracts) [reply src/proposals/20260423/Spell_20260423.t.sol:98]: done

**Review by TheMj0ln1r — CHANGES_REQUESTED** [review]
Fix branch name `dev-1593-spell-24032026` -> `dev-1593-spell-23042026`
